sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -308,19 +308,20 @@ def _workflow_with_pipelinerun(self, task_refs, pipeline, pipeline_template, wor
       }
     }
 
-    # Generate PodTemplate
-    pod_template = {}
+    task_run_spec = []
     for task in task_refs:
       op = pipeline.ops.get(task['name'])
+      task_spec = {"pipelineTaskName":task['name'], "taskPodTemplate": {}}
```

Comment:
nit pick code formatting: add space after colon, maybe break this into a few lines

Reply:
I updated an broke it up to two lines and fixed the missing space.